spotlight-client/src/contentModels/Metric.ts

```diff
@@ -194,24 +220,59 @@ export default abstract class Metric<RecordFormat extends MetricRecord>
     return this.allRecords;
   }
 
+  get unknowns(): Unknowns | undefined {
+    return undefined;
+  }
```

Comment:
I believe this needs be added to the `makeObservable` call in the constructor?

Reply:
Good catch, fixed!